### PR TITLE
Feat/generative ai

### PR DESCRIPTION
**Pull request by neilmistryamplience**
- updated automation files to enable generative AI assistant ready for configuration

### CODE REVIEW

File: content/content-type-schemas/schemas/product-schema.json.hbs [hunk 1]
```diff
@@ -51,6 +51,9 @@
           "language": "json",
           "styles": ".was-price { color: red; }",
           "tools": {
+             "ai":{
```
Comment (rs-amp) [inline content/content-type-schemas/schemas/product-schema.json.hbs:54]: Spacing
```suggestion
            "ai": {
```

File: content/content-type-schemas/schemas/product-schema.json.hbs [hunk 2]
```diff
@@ -51,6 +51,9 @@
...
+              "key":""
```
Comment (rs-amp) [inline content/content-type-schemas/schemas/product-schema.json.hbs:55]: Spacing
```suggestion
              "key": ""
```

File: content/content-type-schemas/schemas/rich-text-schema.json.hbs [hunk 3]
```diff
@@ -18,6 +18,9 @@
           "language": "json",
           "styles": ".was-price { color: red; }",
           "tools": {
+            "ai":{
```
Comment (rs-amp) [inline content/content-type-schemas/schemas/rich-text-schema.json.hbs:21]: Spacing
```suggestion
            "ai": {
```

File: content/content-type-schemas/schemas/rich-text-schema.json.hbs [hunk 4]
```diff
@@ -18,6 +18,9 @@
...
+              "key":""
```
Comment (rs-amp) [inline content/content-type-schemas/schemas/rich-text-schema.json.hbs:22]: Spacing
```suggestion
              "key": ""
```

**Review by rs-amp — APPROVED** [review]
LGTM, small spacing nits

**Review by pixelscript — APPROVED** [review]

**Pull request merged**